src/engine/gui.cpp

```diff
@@ -99,9 +99,9 @@ IMPLOT_API void PlotPieChart(std::map<std::string, T>& map, double x, double y,
 }
 }  // namespace ImPlot
 
-namespace {
-void DrawFlowIcon(ImDrawList* drawList, const ImVec2& a, const ImVec2& b, ax::Drawing::IconType type, bool filled,
-                  ImU32 color, ImU32 innerColor) {
+namespace ax::Drawing {
```

Comment:
This is in the source file so it should be in an anonymous namespace. Also try not to add our own function into other libraries' namespaces.